.pre-commit-config.yaml

```
@@ -14,7 +14,7 @@ repos:
     rev: v1.3.3
     hooks:
     -   id: reorder-python-imports
-        language_version: python2.7
+        language_version: python3.6
```

After trying this, I now get an error:

```bash
~/Projects/le/kolibri update-precommit
.venv ❯ pre-commit run --all-files
[INFO] Installing environment for https://github.com/asottile/reorder_python_imports.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: Command: ('/Users/d/Projects/le/kolibri/.venv/bin/python', '-mvirtualenv', '/Users/d/.cache/pre-commit/repojskl9o3e/py_env-python3.6', '-p', 'python3.6')
Return code: 1
Expected return code: 0
Output: 
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.6'
    
Errors: (none)

Check the log at /Users/d/.cache/pre-commit/pre-commit.log

~/Projects/le/kolibri update-precommit 21s
```

Seems like it's because my virtual environment uses Python 3.8:

```bash
.venv ❯ python --version          
Python 3.8.2
```

I think this line can actually be deleted entirely, and it should just use whatever environment is available:

```suggestion
```

---

The only concern with that is if different language versions produce different import orderings (e.g. changing to Python 3.6 caused some imports to be reordered).

If you use Python 3.8 does it reorder any imports?

---

Switching to 3.8 retains the updated ordering in this PR, so I imagine it's a 2/3 inconsistency.

Another option is just the `python3` alias:

```suggestion
        language_version: python3
```

---

That should work - I think that _should_ also fix the Github action error.